docs/Tutorials/Smart-contracts/Set-up-local-environment.md

```diff
@@ -48,3 +48,9 @@ Then, install `cargo-generate`, which we will need for bootstrapping new CosmWas
 ```sh
 cargo install cargo-generate --features vendored-openssl
 ```
+
+You will also need to install `cargo-run-script` which is needed for optimizing smart contracts. 
```

Comment:
After looking at the rest of the content surrounding this addition, I'd consider saying something more direct, like, "Next, install `cargo-run-script`, which is required to optimize smart contracts.